### PR TITLE
 #165305747 Intergrate Travis CI with readme badge

### PR DESCRIPTION
**Pull request by IssaIan**
#### Description
- Configured Travis CI integration.
- Added travis build badge to readme.md  

#### Type of change
- [ ] Bug fix (nonbreaking change which fixes an issue)
- [ ] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end
- [ ] Integration

#### Checklist:
- [ ] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[165305747](https://www.pivotaltracker.com/n/projects/2326460/stories/165305747)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/andela/ah-the-immortals-backend.svg?branch=develop)](https://travis-ci.org/andela/ah-the-immortals-backend)
```
Comment (Ogutu-Brian) [inline README.md]: Hello @IssaIan ,
Would you make the badge to come after the title of the Project. 
The title gives a short phrase describing what the succeeding section of our project is all about, continuous integration being part of the project.

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IssaIan) [reply README.md]: Done.

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]
Hi @IssaIan ,
Python was not installed during the travis build, would you please fix that so that the builds fail only on the test stage
image

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by matthenge — APPROVED** [review]

**Pull request merged**